src/app/features/genes/components/gene-hero/gene-hero.component.ts

```diff
@@ -103,9 +103,9 @@ export class GeneHeroComponent {
   }
 
   getEnsemblUrl() {
-    if (!this.gene?.ensembl_info || this.gene.ensembl_info.length <= 0)
```

Comment:
Could consider adding a test to confirm that an empty string will be returned when `ensembl_info` is falsy (similar to the test for when `ensembl_info` was an empty array, which was understandably removed in this PR), but up to you!